config_hub/finetune/README.md

```diff
@@ -0,0 +1,10 @@
+## Config files
+
+The table below lists the performances you can expect from the provided config files. Note that you can achieve lower memory consumption by lowering the micro batch size as needed. See the [Dealing with out-of-memory (OOM) errors](../../litgpt/tutorials/oom.md) on lowering the memory requirements.
+
+|                       | Size | Dataset   | Epochs | Val loss | Peak memory | Max seq length | Micro batch size | Precision | Training runtime |
+| --------------------- | ---- | --------- | ------ | -------- | ----------- | -------------- | ---------------- | --------- | ---------------- |
+| tiny-llama/lora.yaml  | 1.1B | Alpaca 2k | 1      | 1.053    | 10.54 GB    | 512            | 8                | bfloat16  | 9.24 min (A10G)  |
+| tiny-llama/qlora.yaml | 1.1B | Alpaca 2k | 4      | 1.074    | 13.32 GB    | 512            | 8                | bfloat16  | 9.89 min (A10G)  |
```

Comment:
did something get mixed up here perhaps?
qlora should have a longer runtime, and use less memory, especially when it is running 4 vs 1 epoch?

Reply:
sorry, full finetuning was 1 epoch (due to overfitting) and QLoRA and LoRA 4 epochs